Kitodo/src/main/java/org/kitodo/production/forms/ImportConfigurationEditView.java

```diff
@@ -225,6 +226,43 @@ public void removeSearchField(SearchField searchField) {
         importConfiguration.getSearchFields().remove(searchField);
     }
 
+    /**
+     * Add given UrlParameter to importConfiguration.
+     *
+     * @param urlParameter UrlParameter to add
+     */
+    public void addUrlParameter(UrlParameter urlParameter) {
+        if (importConfiguration.getUrlParameters().stream().map(UrlParameter::getParameterKey)
+                .collect(Collectors.toList()).contains(urlParameter.getParameterKey())) {
+            Helper.setErrorMessage("URL parameter keys must be unique ('" + urlParameter.getParameterKey()
```

Comment:
The error message is shown, but only showing for the fraction of a second.

Reply:
This seems to have been caused by the `saveButtonToggler`. This hidden button is used to activate the `Save` button and is called whenever the properties of an object being edited - in this case the `ImportConfiguration` - change. And since the `error-messages` component was set to `autoUpdate`, the previously displayed "URL parameter keys must be unique" message would be immediately replaced with an empty message array when the `saveButtonToggler` was triggered. 

To resolve the issue, I replaced the `error-messages` component with `autoUpdate` with an identical `error-messages` component _without_ the `autoUpdate` in the `importConfigurationEdit` view.